release.sh

```diff
@@ -81,9 +81,21 @@ git push origin "$TAG_NAME"
 
 echo "**** Pushing build $BUILD ****"
 
+rm -f $TARBALL.sig install.sh.sig
+
+# Sign the tarball and the install script. Note that we don't sign the channel build number because
+# it wouldn't accomplish anything: If an attacker wanted to provide an old number, they could
+# provide the old signature to match. If an attacker provided a number that hasn't been used
+# before, they would not be able to provide a matching package because no such signed package
+# exists.
+gpg -u 9DA8CCBD --detach-sig $TARBALL
+gpg -u 9DA8CCBD --detach-sig install.sh
+
```

Comment:
might want to do `--digest-algo SHA512` because otherwise i think this defaults to SHA1 (assuming you haven't set algo prefs in gpg.conf)